CRM/Event/Page/Tab.php

```diff
@@ -90,7 +90,7 @@ public function edit() {
     $controller = new CRM_Core_Controller_Simple(
       'CRM_Event_Form_Participant',
       'Create Participation',
-      $this->_action, FALSE, FALSE, TRUE
```

Comment:
The removal of this TRUE has caused the 500 console error that I added it to avoid to come back. The other symptom is the fees not loading.

Reply:
That's weird. I was seeing the opposite - even with `ignoreKey=TRUE` here (PR-branch without df80fba), the AJAX call replied 500 and the fees subform didn't render. With df80fba (with a valid key -- and with `ignoreKey=FALSE`), the AJAX call and the subform worked.

I wonder if there might be some subtle difference in the pageflows or use-cases that we're trying?

Reply:
@totten maybe - I have a good idea of a better fix - but we need to get 'something' out - to be honest I can't understand how the key would be rejected if ignoreKey is false

I am doing a basic search & then selecting some contacts & choosing register participants

I did test that on the pr test site & it failed

Reply:
FWIW, my procedure is very similar, ie

* Open advanced search
* Search for all contacts (no criteria)
* Check the first "Individual" in the demo data ("Adams, Heidi")
* In the task list, choose to register
* For the event, choose any of the three demo events (eg "Rain-forest Cup")

@eileenmcnaughton I can reproduce the error that appears on the test site. (Incidentally, I'm doing this on a different workstaiton/build.) Notably, in both test site and local, the 500 error is not related to qfKey. It's about "getFieldValue failed" (which curiously [arose before](https://github.com/civicrm/civicrm-core/pull/19125#issuecomment-739133215) but had seemed transitory). Here's a backtrace.

<img width="1433" alt="Screen Shot 2020-12-07 at 9 22 41 PM" src="https://user-images.githubusercontent.com/1336047/101443508-714b9b00-38d2-11eb-88e7-96ba3d42cf40.png">

The thing is... to my reading, it's basically saying that [the event-registration form's preProcess()](https://github.com/civicrm/civicrm-core/blob/df80fbabc873519be8b08d2d49a682826e671994/CRM/Event/Page/Tab.php#L121-L130) is applying a mismatched security check. (It's trying to use a `contactId` and `CRM_Contact_Page_View::checkUserPermission()` -- but in the context of a search-task, it doesn't make sense to use a singular `contactId`  for assessing permissions on a bulk action.)

I've pushed up another patch which fixes this symptom for me, but it definitely needs some more eyes to see if it's faithful to contract/permissioning of this route/controller.

Reply:
@totten given that 
1) the person who has reported this has applied the patch in the 5.32 branch & says it is working and
2) we are pretty clear none of this is the final fix

I'm inclined to merge what that person merged & work on the right fix in master

Reply:
The report on 5.32 PR actually pointed to the same problem raised here (just a diff part of the manifestation -- the HTTP 500 error you noted above is the "getFieldValue required" error) - I think they just didn't realize that the screen was misbehaving (because it's a misbehavior of omission; because the immediate error-message is hidden in AJAX console; and the UI-level error message is delayed). So the revision they tested isn't actually working.

Added some more [comments over there](https://github.com/civicrm/civicrm-core/pull/19127#issuecomment-741170965) to try to distinguish 3 issues floating around here. To my testing, the current revision of this branch addresses all 3.